forge-gui/res/cardsfolder/upcoming/mutalith_vortex_beast.txt

```diff
@@ -0,0 +1,11 @@
+Name:Mutalith Vortex Beast
+ManaCost:4 U R
+Types:Creature Mutant Beast
+PT:6/6
+K:Trample
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigFlipCoin | TriggerDescription$ Warp Vortex — When CARDNAME Beast enters the battlefield, flip a coin for each opponent you have. For each flip you win, draw a card. For each flip you lose, CARDNAME Beast deals 3 damage to that player.
+SVar:TrigFlipCoin:DB$ RepeatEach | RepeatPlayers$ Opponent | RepeatSubAbility$ DBCoinFlip
```

Comment:
This will have the same problem like your previous PR
in most cases you can't use Repeat in such way easily because some other abilities care about the amount, in this case _Alms Collector_
You should be able to script around it though, look at _Yusri, Fortune's Flame_ for pointers